config/config.go

```diff
@@ -741,9 +745,10 @@ type MempoolConfig struct {
 	MaxBatchBytes int `mapstructure:"max-batch-bytes"`
 }
 
-// DefaultMempoolConfig returns a default configuration for the Tendermint mempool
+// DefaultMempoolConfig returns a default configuration for the Tendermint mempool.
 func DefaultMempoolConfig() *MempoolConfig {
 	return &MempoolConfig{
+		Version:   MempoolV1,
```

Comment:
Don't we want the default to be `MempoolV0` at least until V1 is complete else the node is just going to panic for anything that uses the default config

Reply:
No, we want v1 to be default. Once I wrap up the 2nd PR, nothing will panic or fail.

Reply:
Ok. So do you plan to merge the PR's together before merging on master?

Reply:
Yes, exactly :)